sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.controller.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable, NotFoundException } from "@nestjs/common";
+import { ApplicationOfferingChangeRequestService } from "../../services";
+import { ApplicationOfferingChangesAPIOutDTO } from "./models/application-offering-change-request.dto";
+import { getUserFullName } from "../../utilities";
+
+@Injectable()
+export class ApplicationOfferingChangeRequestControllerService {
+  constructor(
+    private readonly applicationOfferingChangeRequestService: ApplicationOfferingChangeRequestService,
+  ) {}
+
+  /**
+   * Get the Application Offering Change Request by its id.
+   * @param id the Application Offering Change Request id.
+   * @param options method options:
+   * - `studentId`: student id for student authorization.
+   * - `locationId`: location id for institution authorization.
+   * @returns application offering change request.
+   */
+  async getById(
+    id: number,
+    options?: {
+      studentId?: number;
+      locationId?: number;
```

Comment:
`locationId ` is not used in the method

Reply:
It's required by the `getById` method in `application-offering-change-request.institions.controller`. @ann-aot please let me know if I missed something.

Reply:
but, it is not passed to `getById`, 
![image](https://github.com/bcgov/SIMS/assets/77353155/f56a7c9c-8e52-48f7-b2da-60b799588956)